src/stories/mocks/queries.ts

```diff
@@ -1089,3 +1112,88 @@ export const userPoints = [
         },
     },
 ];
+
+export const mockRecentTrades = [
+    {
+        request: {
+            query: queries.TransactionHistoryDocument,
+            variables: {
+                currency: usdcBytes32,
+                maturity: dec22Fixture.toNumber(),
+                from: -1,
+                to: today,
+                first: 100,
```

Comment:
no first variable in the queries just below? If it is a mistake, call getTransactionQuery() directly.